src/components/fab/EphemeralAudioPlayerFab.tsx

```diff
@@ -0,0 +1,258 @@
+import * as React from 'react';
```

Comment:
I think we don't have this Fab any longer. Can you remove this? We have git if needed, but this code is basically dead